crates/uv/src/commands/pip_compile.rs

```diff
@@ -383,12 +384,30 @@ pub(crate) async fn pip_compile(
         "{}",
         DisplayResolutionGraph::new(
             &resolution,
+            &no_emit_packages,
             generate_hashes,
             include_annotations,
             annotation_style,
         )
     )?;
 
+    // If any "unsafe" packages were excluded, notify the user.
+    let excluded = no_emit_packages
+        .into_iter()
+        .filter(|name| resolution.contains(name))
+        .collect::<Vec<_>>();
+    if !excluded.is_empty() {
+        writeln!(writer)?;
+        writeln!(
+            writer,
+            "{}",
+            "# The following packages were included while generating the resolution:".green()
```

Comment:
Thank you for implementing the feature!

nit: I was confused by "include" here, maybe `The following packages were omitted from the result:` (although maybe omit vs no-emit becomes confusing too, so maybe just excluded)

Reply:
Good idea -- changed in https://github.com/astral-sh/uv/pull/1935.